tests/parquet_test.py

```diff
@@ -13,23 +14,25 @@ def test_parquet(self):
         ak_arr = ak.randint(0, 2**32, SIZE)
         ak_arr.save_parquet("pq_testcorrect", "my-dset")
         pq_arr = ak.read_parquet("pq_testcorrect*", "my-dset")
-        # get the dset from the dictionary in multi-locale cases
+        self.assertTrue((ak_arr ==  pq_arr).all())
+        
         for f in glob.glob('pq_test*'):
             os.remove(f)
-            self.assertTrue((ak_arr ==  pq_arr).all())
 
     def test_multi_file(self):
         adjusted_size = int(SIZE/NUMFILES)*NUMFILES
         test_arrs = []
+        elems = ak.randint(0, 2**32, adjusted_size)
+        per_arr = int(adjusted_size/NUMFILES)
+        print(elems)
```

Comment:
You should take out the `print` here and teh one one line 30 inside the for loop